src/coreclr/inc/switches.h

```diff
@@ -165,3 +165,7 @@
 // If this is uncommented, leaves a file "StubLog_<pid>.log" with statistics on the behavior
 // of stub-based interface dispatch.
 //#define STUB_LOGGING
+
+#ifndef TARGET_WASM
```

Comment:
```suggestion
#ifdef TARGET_WASM
```
I think the idea is to define `PEIMAGE_FLAT_LAYOUT_ONLY` only on wasm.